### DISCUSSION TITLE
Functionality comparison to kubernete ingress-nginx

### DISCUSSION THREAD

**Discussion by sov-msaeed** [nginx/kubernetes-ingress#6920] [General]
Hi, I tried to join slack channel but I couldn't as it seems to refuse my sign-up with any of my mail addresses (even my company one) I dunno what is the problem. is it possible to send me invitation ?

I wanted to join to get real experiences from people who used both ingress-nginx and nginx-ingress-controller form F5. I want to migrate from ingress-nginx and I would like to find out what are functionalities I would miss using the F5 nginx-ingress-controller community version.

**Comment by jjngx** [comment]
Hi @sov-msaeed , have you tried `nginxcommunity.slack.com`? This is the F5/NGINX community slack. You are also more than welcome to join our community video calls (see the schedule in the README file).

**Reply by sov-msaeed** [reply]
I don't know how to sign up. I assume i need to sign up here https://community.nginx.org/joinslack

**Reply by jjngx** [reply]
`https://community.nginx.org/login`

**Reply by jjngx** [reply]
and then use the signup flow

**Reply by sov-msaeed** [reply]
yes I tried that. and i got this error
```
 Email is not allowed from that email provider. Please use another email address.
```
I used all mails possible, also my company mail address.

**Comment by jjngx** [comment]
@sov-msaeed could you please try again and let me know results?

**Reply by sov-msaeed** [reply]
Unfortunately same issue.

**Reply by jjngx** [reply]
@sov-msaeed thanks! We are working on this issue. It's not on your side. I will let you know as soon as I have updates.

**Reply by jjngx** [reply]
@sov-msaeed , could you please try [this direct invitation link](https://join.slack.com/t/nginxcommunity/shared_invite/zt-2l8ugu1gl-pdOzdY_13UM4NmqOhl1mEQ)?

**Reply by sov-msaeed** [reply]
Works now. thanks.

**Reply by jjngx** [reply]
Perfect!
